deploy/factory.ts

```diff
@@ -0,0 +1,139 @@
+        ),
+        minCheckpointDuration: BigInt(dayjs.duration(24, "hours").asSeconds()),
+        maxCheckpointDuration: BigInt(dayjs.duration(24, "hours").asSeconds()),
+        minPositionDuration: BigInt(dayjs.duration(7, "days").asSeconds()),
+        maxPositionDuration: BigInt(dayjs.duration(30, "days").asSeconds()),
+        minFixedAPR: parseEther("0.0100"),
+        maxFixedAPR: parseEther("0.6000"),
+        minTimeStretchAPR: parseEther("0.0100"),
+        maxTimeStretchAPR: parseEther("0.6000"),
+        minFees: {
+          curve: parseEther("0.001"),
+          flat: parseEther("0.0001"),
+          governanceLP: parseEther("0.1500"),
+          governanceZombie: parseEther("0.0300"),
+        },
+        maxFees: {
+          curve: parseEther("0.01"),
+          flat: parseEther("0.001"),
+          governanceLP: parseEther("0.1500"),
+          governanceZombie: parseEther("0.0300"),
+        },
+        linkerFactory: linkerFactory.address,
+        linkerCodeHash,
+      },
+      "factory",
+    ],
+  });
+  const erc4626Core = await deploy("ERC4626HyperdriveCoreDeployer", {
+    from: deployer,
+  });
+  const erc4626Target0Deployer = await deploy("ERC4626Target0Deployer", {
+    from: deployer,
+  });
+  const erc4626Target1Deployer = await deploy("ERC4626Target1Deployer", {
+    from: deployer,
+  });
+  const erc4626Target2Deployer = await deploy("ERC4626Target2Deployer", {
+    from: deployer,
+  });
+  const erc4626Target3Deployer = await deploy("ERC4626Target3Deployer", {
+    from: deployer,
+  });
+  const erc4626Target4Deployer = await deploy("ERC4626Target4Deployer", {
+    from: deployer,
+  });
+  const erc4626Coordinator = await deploy(
+    "ERC4626HyperdriveDeployerCoordinator",
+    {
+      from: deployer,
+      args: [
+        hyperdriveFactory.address,
+        erc4626Core.address,
+        erc4626Target0Deployer.address,
+        erc4626Target1Deployer.address,
+        erc4626Target2Deployer.address,
+        erc4626Target3Deployer.address,
+        erc4626Target4Deployer.address,
+      ],
+    },
+  );
+
+  const stethCore = await deploy("StETHHyperdriveCoreDeployer", {
+    from: deployer,
+  });
+  const stethTarget0Deployer = await deploy("StETHTarget0Deployer", {
+    from: deployer,
+  });
+  const stethTarget1Deployer = await deploy("StETHTarget1Deployer", {
+    from: deployer,
+  });
+  const stethTarget2Deployer = await deploy("StETHTarget2Deployer", {
+    from: deployer,
+  });
+  const stethTarget3Deployer = await deploy("StETHTarget3Deployer", {
+    from: deployer,
+  });
+  const stethTarget4Deployer = await deploy("StETHTarget4Deployer", {
+    from: deployer,
+  });
+  const lido =
+    hre.network.name == "sepolia"
+      ? (
+          await deploy("MockLido", {
+            from: deployer,
+            args: [parseEther("0.035"), ADMIN, true, parseEther("500")],
+            log: true,
+          })
+        ).address
+      : LIDO_ADDRESS;
```

Comment:
Shouldn't we be deploying a mock vault for the ERC4626 instances on Sepolia too? I get that we don't need it to instantiate the coordinator, but we'll still need to deploy it eventually.